pkgs/development/libraries/libev/default.nix

```diff
@@ -7,6 +7,11 @@ stdenv.mkDerivation rec {
     url = "http://dist.schmorp.de/libev/${name}.tar.gz";
     sha256 = "1jyw7qbl0spxqa0dccj9x1jsw7cj7szff43cq4acmklnra4mzz48";
   };
+
+  # Version 4.19 is not valid C11 (which Clang default to)
+  # Check if this is still necessary on upgrade
+  NIX_CFLAGS_COMPILE = stdenv.lib.optionalString stdenv.cc.isClang "-std=c99";
```

Comment:
This use of `optionalString` is going to cause a mass-rebuild on linux, sadly. `optionalString` behaves like

```
optionalString b s = if b then s else ""
```

and the empty string gets put into the derivation, whereas in

```
if b then s else null
```

the `null` is filtered out, leaving the linux derivation unchanged :(